corehq/apps/receiverwrapper/views.py

```diff
@@ -138,6 +141,7 @@ def _process_form(request, domain, app_id, user_id, authenticated,
         try:
             result = submission_post.run()
         except XFormLockError as err:
+            logger.warning('Unable to get lock for form %s' % err)
```

Comment:
nit: inconsistent with other logging calls in this PR.
```suggestion
            logger.warning('Unable to get lock for form %s', err)
```